fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -239,13 +304,40 @@ class DemographicParity(UtilityParity):
     The :meth:`signed_weights` method will compute the costs according
     to Example 3 of
     `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
+
+    This Moment also supports control features, which can be used to
```

Comment:
I'd suggest staying away from the term `Moment` here as well, unless you link to the base class using ReST

Reply:
Added link